src/kirby/components/divider/divider.component.scss

```diff
@@ -12,4 +12,11 @@
 hr {
   border: 0;
   border-top: 1px solid var(--kirby-divider-color, get-color('medium'));
+
+  &.no-margin {
+    margin-top: 0;
```

Comment:
I'm pretty sure that `margin-top` and `margin-block-start` is "the same" (that same goes for `margin-bottom` and `margin-block-end`?

Furthermore, we should have an explicit size variable, setting the `margin-top|bottom` when the margin is not disabled. As it is now, the user agent stylesheet simply bleeds through, and will result in a different margin size between browsers. Consider using the kirby `size`-function.

Reply:
Fixed 👍